src/libraries/System.Reflection.Emit/tests/PersistableAssemblyBuilder/AssemblySaveILGeneratorTests.cs

```diff
@@ -1592,115 +1591,109 @@ public void DeeperNestedTryCatchFilterFinallyBlocks()
 
         private static int Int32Sum(int a, int b) => a + b;
 
-        [ConditionalFact(typeof(RemoteExecutor), nameof(RemoteExecutor.IsSupported))]
+        [Fact]
         public void EmitCalliBlittable()
         {
-            RemoteExecutor.Invoke(() =>
```

Comment:
Why was the use of remote executor changed?

Reply:
Now using collectable ALC instead of default, the discussion is here https://github.com/dotnet/runtime/pull/96805#discussion_r1448108301
https://github.com/dotnet/runtime/blob/e6d93336bd60870150cd9aad1c8c90fbe6fda496/src/libraries/System.Reflection.Emit/tests/PersistableAssemblyBuilder/AssemblySaveTools.cs#L12-L22